.github/workflows/e2e-tests.yml

```
@@ -15,7 +15,51 @@ on:
 jobs:
+  publish-e2e-registry:
+    name: Publish E2E registry
+    runs-on: starsling-ubuntu-24.04
+    timeout-minutes: 30
+    permissions:
+      contents: read
+    env:
+      TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+      TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
+      TURBO_CACHE: remote:r
+      MASTRA_E2E_REGISTRY_DIR: ${{ runner.temp }}/mastra-e2e-registry
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)